.github/CODEOWNERS

```diff
@@ -23,6 +23,9 @@
 # PRLabel: %App Configuration
 /sdk/appconfiguration/                          @annelo-msft @AlexanderSher
 
+/sdk/applicationinsights/Microsoft.Azure.App*/  @divya-jay @geneh @alongafni
```

Comment:
@geneh recommend you just set this for /sdk/applicationinsights, I can't imagine anything in here that you probably don't want awareness of.

Reply:
@mitchdenny AFAIK we do not own the other library at /sdk/applicationinsights/Microsoft.Azure.Management.ApplicationInsights/